### PR TITLE
feat(ci): Add create release action and deploy action

### PR DESCRIPTION
**Pull request by adrianmjim**

### CODE REVIEW

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        run: |
+          sudo apt-get install moreutils
+          jq '.version = "${{ steps.version.outputs.VERSION }}"' packages/nestjs-firebase-admin/package.json | sponge packages/nestjs-firebase-admin/package.json
+          
+      - name: Commit release
+        run: |
+          git add .
+          git commit --amend --no-verify --no-edit
+          git push --force
+          git push --tags
```
Comment (notaphplover) [inline .github/workflows/create_release.yml]: Ohh I see :)

What if we do not ammend this commit and instead push a new commit:

```suggestion
      - name: Commit release
        run: |
          git add .
          git commit --no-verify -m "chore: bump version numbers"
          git push
          git push --tags
```

File: turbo.json [hunk 2]
```diff
@@ -23,6 +23,10 @@
       "dependsOn": ["^build"],
       "inputs": ["src/**/*.ts"]
     },
+    "publish:package": {
+      "cache": false,
```
Comment (notaphplover) [inline turbo.json:27]: :heart:

**Review by notaphplover — CHANGES_REQUESTED** [review]
Looks great! I just prefer to avoid changing git history in favor of writting an additional commit.

File: .github/workflows/create_release.yml (hunk 1 above)
Comment (adrianmjim) [reply .github/workflows/create_release.yml]: I am a fan of not amending commits, I love to maintain an explicit git history. But this case is a bit special. standard-version is increasing the version and creating a tag with the new version. We are adding new file changes that we want to include in the release that has already been tagged. Therefore, if we make a new commit, all the changes we just made will be outside of the release that has just been tagged.

Comment (notaphplover) [reply .github/workflows/create_release.yml]: Can we commit this change before appliying the tag? That would be the best

Comment (adrianmjim) [reply .github/workflows/create_release.yml]: Changed :smile:

**Review by notaphplover — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**